docs/website/packages/codegen/copy.md

```diff
@@ -0,0 +1,46 @@
+---
+sidebar_position: 2
+title: Copying
+description: codegen's copy function
+---
+
+The `copy` function is designed to facilitate the duplication of source code from one FHIR version to another.
+
+## Basic Example
+
+### Set Up Your Source and Target Directories
+
+Ensure that you have two directories: one for the source FHIR version and another for the target FHIR version. For example, you might have a directory structure like this:
+
+- `src/r4b` (source)
+- `src/r5` (target)
+
+### Prepare Conditional Preprocessing
+
+If you need conditional preprocessing (such as including code only for specific FHIR versions), add preprocessing directives to your source files. For example:
+
+```ts
+// #if fhir >= r5
+// Specific code for FHIR version r5 or higher
+```
+
+### Create a Copy Script
+
+Next, add a script to your `package.json` to use the `copy` function. Here's an example:
```

Comment:
I think adding a script is up to the discretion of the author. We should document it as a command-line tool.

Suggestion:

`npx -y @bonfhir/codegen copy --source-fhir r5 --target-fhir r4b --source "./r5" --target "./r4b"`

Also - we should mention that the tool expect [`prettier`](https://prettier.io/) to be present in the environment as it runs it after altering the files.